keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -315,6 +315,7 @@ extensibility in the future, the current pod specs will still work.
 4.	Identify minimal in-cluster operational core (from existing CPU Manager, Memory Manager, Topology Manager)
 5.  Replace CCI policy with a component integrated in None and Static CPU Manager policies
 6.	E2E testing including other components
+7.  Defragmentation improvements of resources.
```

Comment:
I would define "defragmentation" and what the implications are - why is it important and so forth

Reply:
Added.